components/mixins/experience-event/experienceevent-service-payment-details.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-service-payment-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Service Payment Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["financial_services", "telecom"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Service Payment Details such as ID, type, and amount.",
+  "definitions": {
+    "service-payment-details": {
+      "properties": {
+        "xdm:": {
```

Comment:
object name missing

Reply:
The name of the property is still missing here. We can't have just "xdm:" xdm is just a name space, we need to have a property name here too.

Reply:
Sorry about that.  It was a build issue with the underlying macro - and that's been fixed now.